qa/objectstore/bluestore.yaml

```diff
@@ -20,3 +20,21 @@ overrides:
         osd failsafe full ratio: .95
 # this doesn't work with failures bc the log writes are not atomic across the two backends
 #        bluestore bluefs env mirror: true
+  ceph-deploy:
+    fs: xfs
+    bluestore: yes
```

Comment:
@tmuthamizhan it has to be a separate test, test1 + bluestore.yaml and test1 + filestore.yaml, cant mix both configs in same test, so the suite will have to permute with that options within that folder.